Makefile

```diff
@@ -66,4 +78,4 @@ dist:
 	github-release upload --tag ${TAG} --name wr-macos-x86-64.zip --file darwin-dist.zip
 	@rm -f wr linux-dist.zip darwin-dist.zip
 
-.PHONY: build test race lint lintextra install clean dist
+.PHONY: build test race lint lintextra install clean dist compile_k8s_tmp test-e2e
```

Comment:
and test-k8s-unit? clean should also remove /tmp/wr

Reply:
Done